vaadin-messages-flow-parent/vaadin-messages-flow/src/main/java/com/vaadin/flow/component/messages/MessageList.java

```diff
@@ -40,9 +42,10 @@
  * @author Vaadin Ltd.
  */
 @Tag("vaadin-message-list")
+@JsModule("./messageListConnector.js")
 @JsModule("@vaadin/vaadin-messages/src/vaadin-message-list.js")
 @NpmPackage(value = "@vaadin/vaadin-messages", version = "1.0.0-alpha1")
-public class MessageList extends Component implements HasStyle, HasSize {
+public class MessageList extends Component implements HasStyle, HasSize, LocaleChangeObserver {
```

Comment:
This file is not formatted by the Vaadin code conventions. This line exceeds the max line length.

I recommend using the `Save Actions` plugin in IntelliJ and configure it to reformat file and optimize imports on save.

Reply:
It was in `Edit > Code Style > Java > Ensure right margin is not exceeded`, not sure why it didn't wrap with `Hard wrap at: 80` already set 😅

Reply:
So it's still a bit unclear to me: did you get the auto-formatting to work? With my settings, it also wraps the `executeJs` line.

Reply:
Seems like it doesn't automatically wrap long strings (or I didn't configure it correctly). I did it manually for now, I'll look for a setting!

Reply:
Yeah, it doesn't wrap strings and I think it's fine.

But it starts the string from a new line now, which was the important thing. :) Because now my auto-formatter doesn't make that change when I'm working on something un-related.